src/WordCard.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const WordCard = ({ data, handleClick }) => {
+    return(
+      <div className="card" value={data} onClick={(e) => handleClick(e, data)}>  
+        {data.word}    
+      </div>  
+    )   
+};
+
```

Comment:
Can someone point me to documentation on the way this is written? I thought this would be a component but it's rendering the cards so that bit seems to be working and I think I just don't know what this does.

Also would be great if someone could explain to me where "data" is coming from?

Reply:
So, I think you are asking about React classes and presentational components.

React classes are the types you may be familiar with, 
```
class WordCard extends React.Component {
}
```

However, if you know your react component will just be accepting data through PROPS, without much data changing or state management, then we can just render a presentational component styled as above:
```
const WordCard = (props) => {
// simplified component that just renders something based on data passed in
}
```
This is because the React Class type components isnt necessary, we don't need an entire Class type with numerous state management and methods when a function component is all thats needed.

Does that answer your question?

Reply:
Ah I see...I didn't know this existed. Thanks!